modules/authorization-xacml/src/main/java/org/opencastproject/authorization/xacml/XACMLAuthorizationService.java

```diff
@@ -506,15 +504,11 @@ private PolicyDecisionPoint getPolicyDecisionPoint(File xacmlFile) {
     sb.append("</ns:Location></ns:Policy></ns:Policies><ns:Locators>");
     sb.append("<ns:Locator Name=\"org.jboss.security.xacml.locators.JBossPolicyLocator\">");
     sb.append("</ns:Locator></ns:Locators></ns:jbosspdp>");
-    InputStream is = null;
-    try {
-      is = IOUtils.toInputStream(sb.toString(), "UTF-8");
+    try (InputStream is = IOUtils.toInputStream(sb.toString(), "UTF-8")) {
       return new JBossPDP(is);
     } catch (IOException e) {
       // Only happens if 'UTF-8' is an invalid encoding, which it isn't
       throw new IllegalStateException("Unable to transform a string into a stream");
-    } finally {
-      IOUtils.closeQuietly(is);
```

Comment:
`IOUtils.closeQuietly(…)` is deprecated and we can simply use a try-with-resources statement